test/unit/filters/userStatusFilterSpec.js

```diff
@@ -0,0 +1,127 @@
+/* global describe, it, inject, expect, beforeEach , tsNow*/
+
+describe('userStatus filter', function () {
+  var $filter, _, userStatusFilter
```

Comment:
Today I've learned that you can share variables by assigning them to `this`.

See above for explanation.